src/enclave/enclave.h

```diff
@@ -19,6 +19,17 @@
 #include "rpcmap.h"
 #include "rpcsessions.h"
 
+// This is a hack to avoid recompiling libloggingenc.so:
+// A binary that exposes the same API as the current one, yet produces
+// a different signature, is needed so that code upgrade can be tested.
+// The quick way to do that is to copy the binary and modify a few
+// bytes on the text section of the new binary. This function produces
+// a few bytes that can be safely overwritten.
+extern "C" __attribute__((visibility("default"))) int stub_for_code_signing()
```

Comment:
I'm wary of having this in every enclave. Can we at least hide it with an `#ifdef`, and perhaps move it to a separate file only built with `liblogging`?